sync/statesync/sync_test.go

```diff
@@ -260,7 +260,7 @@ func TestResumeSyncAccountsTrieInterrupted(t *testing.T) {
 		GetLeafsIntercept: intercept.getLeafsIntercept,
 	})
 
-	require.Equal(t, uint32(2), intercept.numRequests)
+	require.LessOrEqual(t, uint32(2), intercept.numRequests.Load())
```

Comment:
At a glance, I thought "less than or equal" "2" "numRequests" and incorrectly thought that the thing under test needed to be leq 2.
```suggestion
	require.GreaterOrEqual(t, intercept.numRequests.Load(), uint32(2))
```
Since we actually want the thing under test to be at _least_ 2, should we be using `GreaterOrEqual`?